scale/docs/architecture/index.rst

```diff
@@ -11,6 +11,7 @@ Architecture
    overview
    jobs/index
    logs
+   messaging
```

Comment:
This new app needs to be added to appropriate index.rst doc file for generated code docs.